README.md

```diff
@@ -259,11 +259,28 @@ Kubernetes cluster.
 ```shell script
 kubectl logs -f -n app-prod app-form-web-app-1111111111-aaaaa
 ```
+
 ### ACP integration
 
-The current target MVP checks only the decision date eligibility before passing the user to an existing external service 
+The current target MVP checks only the decision date eligibility before passing the user to an existing external service
 Appeals Casework Portal (ACP) for user to submit their appeal.
 In the FWA ui app this behaviour is controlled by setting the environment variable `SERVER_LIMITED_ROUTING_ENABLED`.
 For convenience, a placeholder environment variable is present in the `docker-compose.yml` file.
 With this set to `true` only a limited set of pages are accessible and user will be taken to ACP from decision date eligibility check.
 With this set to `false` all existing pages will be accessible and user will proceed to LPD selection instead of ACP.
+
+### Packages Logging
+
+See `*/src/lib/logger.js`
+
+Package applications use [Pino](http://getpino.io). All logs are sent to the
+STDOut so they can be picked up when deployed by the Kubernetes pod(s). It
+is important to use the log levels (trace, debug, info, warn, error, fatal)
+appropriately so that filtering can be done during production.
+
+Also installed is `express-pino-logger` which add `req.log` to the incoming
+HTTP request. This automatically logs incoming HTTP requests and apply a unique
+correlation ID to make tracing errors through easier. For the most part, this
+should be the instance used for logging.
+
+Good practice for logging statements: @todo
```

Comment:
Rather than putting this as a `@todo` in the code, can we actually get this done as part of this PR?

Reply:
Done.